Significantly offload scoring sections of the `Game mode` articles (#6607)
ppy/osu-wiki: wiki/Game_mode/osu!mania/en.md

## Scoring

> [The osu!mania score](/wiki/Gameplay/Score/osu!mania) is capped at 1 million and consists of two parts, each contributing 50% of total score:

Reviewer: ```suggestion
[Score in osu!mania](/wiki/Gameplay/Score/osu!mania) is capped at 1 million and consists of two parts, each contributing 50% of the total score:
```

> 1. Base score, which is [judgement](/wiki/Gameplay/Judgement/osu!mania)-centered. Judgement determines an object's base scoring value (MAX, 300, 200, 100, 50, or 0 in case of a miss).

Reviewer: ```suggestion
1. Base score, which is [judgement](/wiki/Gameplay/Judgement/osu!mania)-centred. Judgement determines a hit object's base scoring value (MAX, 300, 200, 100, 50, or 0 in case of a miss).
```

Author: > centred

ಠ_ಠ

> 2. Bonus score, which is based on judgement and a floating bonus multiplier, which increases with a rainbow 300 or 300, and decreases with a 200 or below.

Reviewer: ```suggestion
2. Bonus score, which is based on judgement and a floating bonus multiplier, which increases when getting a 300 or rainbow 300, and decreases with a 200 or below.
```

Author: accepting for now, but I'd rather use `MAX`

> 2. Bonus score, which is based on judgement and a floating bonus multiplier, which increases when getting a 300 or rainbow 300, and decreases with a 200 or below.

Reviewer: > accepting for now, but I'd rather use MAX

oh yea you're right, just use that

```suggestion
2. Bonus score, which is based on judgement and a floating bonus multiplier, which increases when getting a MAX or a 300, and decreases with a 200 or below.
```

```
Score = BaseScore + BonusScore

BaseScore = (MaxScore * ModMultiplier * 0.5 / TotalNotes) * (HitValue / 320)

BonusScore = (MaxScore * ModMultiplier * 0.5 / TotalNotes) * (HitBonusValue * Sqrt(Bonus) / 320)
Bonus = Bonus before this hit + HitBonus - HitPunishment / ModDivider
Bonus is limited to [0, 100], initially 100.

MaxScore = 1 000 000
ModMultiplier = The score multiplier of the selected mods (difficulty reduction and/or nK)
ModDivider = The punishment divider of the selected mods (difficulty increase)

Judgement  HitValue  HitBonusValue  HitBonus  HitPunishment
   MAX       320          32            2
   300       300          32            1
   200       200          16                        8
   100       100           8                       24
    50        50           4                       44
  Miss         0           0                        ∞

       Mod  ModMultiplier  ModDivider
      Easy       0.5
    NoFail       0.5
  HalfTime       0.5
  HardRock                    1.08
DoubleTime                     1.1
 NightCore                     1.1
    FadeIn                    1.06
    Hidden                    1.06

> Flashlight                    1.06

Reviewer: just a note for the future that ModDivider is [not a thing anymore](https://www.twitch.tv/videos/1237832029?t=02h19m07s)

> After the beatmap is over, the score is assigned a [grade](/wiki/Gameplay/Grade#osu!mania), a short accuracy assessment in form of a single letter. A golden or silver SS denotes 100% accuracy.

Author: note to self: rewrite according to other articles
